packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -181,6 +181,26 @@ function flattenTree( input = {}, prefix, token ) {
 	return result;
 }
 
+/**
+ * Resolves ref into the value it is pointing to.
+ *
+ * @param {Object} refObj      The reference we are pointing to.
+ *
+ * @param {string} blockStyles Block styles.
+ *
+ * @return {string} The resolved referenced value.
+ */
+function resolveRefValue( refObj, blockStyles ) {
+	if ( ! refObj.ref ) {
+		return refObj;
+	}
```

Comment:
Is there a benefit to this function knowing about the object that contains the `ref`? Would it be making the function simpler to just pass the ref only?

Reply:
mmh, I see what you mean, we might not need a function for this at all, it will end up being two lines, I'm not sure if we need the abstraction or if it will need to be reused at some point in time